src/main/java/nl/knaw/dans/easy/DataciteServiceException.java

```diff
@@ -18,12 +18,18 @@
 public class DataciteServiceException extends Exception {
 
     private static final long serialVersionUID = 1L;
+    protected int httpStatusCode = 0;
```

Comment:
do you really need this one to be `protected`? Can't it be private.
The only reason for making it `protected` is the `DataciteUserErrorException`, but that one already sets the correct `httpStatusCode` in calling `super(..., ...)`.

Reply:
You are right, changed it back to private